wagtail/contrib/typed_table_block/static_src/typed_table_block/scss/typed_table_block.scss

```diff
@@ -42,6 +48,10 @@
 
         td {
             vertical-align: top;
+            min-width: 320px;
```

Comment:
I understand why it was necessary to do this for now, but it would be really cool in the future if the min-width could be defined per-column in the TypedTableBlock definition.